synapsis/src/main/java/com/soen/synapsis/appuser/AppUserController.java

```diff
@@ -46,9 +46,9 @@ public String getAdminData() {
         return "This is the admin page";
     }
 
-    @PutMapping("/company/setCandidateToRecruiter")
+    @PutMapping("/company/markCandidateToRecruiter")
     @ResponseBody
-    public String markCandidateToRecruiter(AppUser appUser, @AuthenticationPrincipal AppUserDetails companyUser) {
+    public String markCandidateToRecruiter(AppUser appUser, @AuthenticationPrincipal AppUser companyUser) {
```

Comment:
Why did this have to be changed?

Reply:
Due to the JPA relationships. The company is an AppUser.